package.json

```diff
@@ -42,7 +42,7 @@
         "node-notifier": "^10.0.1",
         "teraslice-test-harness": "^0.29.0",
         "ts-jest": "^29.1.1",
-        "tslib": "^2.6.1",
+        "tslib": "2.6.2",
```

Comment:
Do we really want to remove the leading `^` in these?

Reply:
No I did not realize that when I upgraded the package that it removed the ```^```. I added it back 
Commit with change ref: https://github.com/terascope/standard-assets/pull/721/commits/173c5c17726162cc9f9c7159126aa0f596fa9909